test/data/manifests/fedora-coreos-container.mpp.yaml

```diff
@@ -89,6 +89,7 @@ pipelines:
             sysroot:
               readonly: false
               bootloader: none
+              bls-append-except-default: grub_users=""
```

Comment:
run osbuild-mpp and update the `fedora-coreos-container.json` file
when you open the PR upstream